README.md

```diff
@@ -205,6 +205,20 @@ To run the Jest test suite, execute the command `docker exec -it webpack_watcher
 
 To update snapshots, execute `docker exec -it webpack_watcher npm run-script update-snapshot`
 
+## Data Validation Testing
+
+Data validation scripts are in scripts/data_validation folder. To run the data validation scripts, follow the steps below:
+
+1. copy env.json file from env_sample.json file, and add configuration values:
+   ```sh
+   cp scripts/data_validation/env_sample.json scripts/data_validation/env.json
+   ```
+2. run data validation script within docker container:
+
+   ```sh
+   docker exec -it student_dashboard /bin/bash -c "python scripts/data_validation/<script file>"
```

Comment:
Rather than having a placeholder, I'd probably have this be specific like the following. I'm guessing you had it like this because additional scripts might be added? We can make these specific too. 


* Validate Expanded Table - This script will validate queries against the expanded and regular events table are identical.

```sh
   docker exec -it student_dashboard /bin/bash -c "python scripts/data_validation/events_vs_expanded.py"
```

Reply:
I'd add a note to this readme that the queries hard-coded in these files have Michigan specific filters so they may have to be modified to be used for other schools unless we make them more configurable.